protected/humhub/modules/admin/views/layouts/module-old.php

```diff
@@ -0,0 +1,15 @@
+<?php $this->beginContent('@admin/views/layouts/main.php') ?>
```

Comment:
Can we remove this file?

Reply:
It is still used for url `/marketplace/purchase`

Reply:
@yurabakhtin See here: https://github.com/humhub/humhub/pull/5476#discussion_r782399237

Reply:
Done.